paddle/scripts/docker/jupyter_notebook_config.py

```diff
@@ -0,0 +1,27 @@
+# from https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/docker/jupyter_notebook_config.py
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import os
+from IPython.lib import passwd
+
+c.NotebookApp.ip = '*'
+c.NotebookApp.port = int(os.getenv('PORT', 8888))
```

Comment:
Done. Also use `git rebase` purge this config file in git history.

I'm building an image base on this commit and test it.